geoengine/workflow.py

```diff
@@ -329,34 +341,44 @@ def __get_wcs_tiff_as_memory_file(self, bbox: QueryRectangle, timeout=3600) -> r
 
         return memory_file
 
-    def get_array(self, bbox: QueryRectangle, timeout=3600) -> np.ndarray:
+    def get_array(self, bbox: QueryRectangle, timeout=3600, force_no_data_value=None) -> np.ndarray:
```

Comment:
can you add a type annotation?

Reply:
[82fc21d](https://github.com/geo-engine/geoengine-python/pull/60/commits/82fc21da4cf428ac6c02945b0a1d010245431c27)